packages/services/f-test-data-creator/src/clients/publicApi.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable camelcase */
+const axios = require('axios');
+const { getLanguageForTenant } = require('../configuration/tenants');
+
+module.exports = class PublicApiService {
+    constructor (configuration) {
+        this.createUserUrl = configuration.Services.Public.CreateUserUrl;
+        this.authorizationUrl = configuration.Services.Public.AuthorizationUrl;
+        this.token = configuration.Services.Public.AuthorizationToken;
+    }
+
+    async createConsumer (userInfo, timeout = 5000) {
+        const config = {
+            headers: {
+                'Content-Type': 'application/json',
+                Authorization: this.token
```

Comment:
Ditto

Reply:
bizarrely I always get linting issues whenever I turn Authorization into a string - same thing happened in my last PR for the checkout API services. I can ignore the linting issues though and change it :)

Reply:
I left a comment on this above — I think we should leave it as is.